packages/flutter_tools/pubspec.yaml

```diff
@@ -46,32 +46,29 @@ dependencies:
   test_core: 0.2.5
 
   # Code generation dependencies
-  build_runner_core: 3.1.1
+  build_runner_core: 4.0.0
```

Comment:
See: https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/commands/update_packages.dart#L22

Reply:
This needs to include all of the build_* versions

Reply:
That works! Please take another look on the new changes.